tests/test_F0_output.py

```diff
@@ -0,0 +1,189 @@
+"""
+- Aggregate demand profiles to a total demand profile
+- Plotting all energy flows for both 14 and 365 days for each energy bus
+- Store timeseries of all energy flows to excel (one sheet = one energy bus)
+- execute function: plot optimised capacities as a barchart (F1)
+- execute function: plot all annuities as a barchart (F1)
+- store scalars/KPI to excel
+- process dictionary so that it can be stored to Json
+- store dictionary to Json
+"""
```

Comment:
```suggestion
- Aggregate demand profiles to a total demand profile
- Plotting all energy flows for both 14 and 365 days for each energy bus
- Store timeseries of all energy flows to excel (one sheet = one energy bus)
- Execute function: plot optimised capacities as a barchart (F1)
- Execute function: plot all annuities as a barchart (F1)
- Store scalars/KPI to excel
- Process dictionary so that it can be stored to Json
- Store dictionary to Json
"""
```

Reply:
I discovered that one can select more than one line :)